Assets/Bullet.cs

```diff
@@ -28,6 +26,13 @@ void FixedUpdate()
 
     void OnTriggerEnter2D(Collider2D other)
     {
-        //other.gameObject.GetComponent<Health>().Damage(blah);
+        Health health = other.gameObject.GetComponent<Health>();
+
+        // check whether the colliding object can be damaged
+        if (health)
+        {
+            health.Damage(10); // TODO: damage will vary based on player and enemy?
```

Comment:
Yeah, we should probably add a damage variable to the bullet class that can be set in the editor for different bullet prefabs. No worries for this PR though.